<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categoria_form</class>
 <widget class="QWidget" name="categoria_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 53, 79);
border: 4px solid #00aaff; /* Contorno neon azul */</string>
  </property>
  <widget class="QPushButton" name="categorySaveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 103, 155);
color: rgb(255, 255, 255);
border-radius: 6px;
border: 0px solid black;</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="category_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 6px;
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border: 0px solid #00aaff; /* Contorno neon azul */
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Categoria:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="categoryCancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 103, 155);
color: rgb(255, 255, 255);
border-radius: 6px;
border: 0px solid black;</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border: 0px</string>
   </property>
   <property name="text">
    <string>Nueva Categoria</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
